packages/contracts-bedrock/.gitignore

```diff
@@ -1,22 +1,17 @@
+# Artifacts and Cache
 artifacts
 forge-artifacts
 cache
+
+resource-metering.csv
+
 typechain
 coverage.out
-.deps
 broadcast
-genesis.json
-src/contract-artifacts.ts
-tmp-artifacts
-deployments/mainnet-forked
 deploy-config/mainnet-forked.json
 test-case-generator/fuzz
-.resource-metering.csv
 scripts/differential-testing/differential-testing
-deployments/900
-deployments/901
```

Comment:
we want to bring back the deployments here, 900,901,hardhat,getting-started
also deployments/*/.deploy

Otherwise it will leave artifacts for devnet/tutorial in git

We can delete `deploy-config/mainnet-forked.json`